sparta/sparta/utils/SpartaSharedPointer.hpp

```diff
@@ -134,11 +136,9 @@ namespace sparta
          * nullptr with no references
          */
         SpartaSharedPointer(SpartaSharedPointer && orig) :
-            memory_block_(orig.memory_block_),
             ref_count_(orig.ref_count_)
         {
             // DO NOT unlink.
-            orig.memory_block_ = nullptr;
             orig.ref_count_    = nullptr;
```

Comment:
There are a few places where the spacing needs to be fixed after removing `memory_block_`.

Reply:
Done.  Thanks!